README.rst

```diff
@@ -1741,6 +1741,72 @@ If the request is made without passing the ``time`` argument, the server local t
 
 The ``time`` parameter was added to support `resilient collection and sending of data by the OpenWISP Monitoring Agent <https://github.com/openwisp/openwrt-openwisp-monitoring#collecting-vs-sending>`_.
 
+List wifi session
+#################
```

Comment:
mention available filters in the GET endpoints please

Reply:
Sure, I'll mention :+1: